src/api/docker.ts

```diff
@@ -255,3 +259,40 @@ export function isDockerInstalled(): Observable<boolean> {
     });
   });
 }
+
+export function calculateContainerStats(
```

Comment:
fix that style and remember how we handle that:
```ts
export function calculateContainerStats(
  container: dockerode.ContainerInfo,
  processes: any
): Promise<any> {
  return docker.getContainer(container.id).stats()
    .then(stream => { ....

    });
}
```
then please keep using that style in the future.